src/quests/cognac/tasks/round.ts

```diff
@@ -13,6 +13,14 @@ export class Round {
 
   getTasks(): Task[] {
     return [
+      {
+        name: "Initial wait",
+        completed: () => get(CURRENT_STENCH) !== "",
```

Comment:
Will this task ever complete if we are running solo?

Reply:
Ah good call. Will number of CURRENT_PLAYERS be accurate at this point? Could add `|| CURRENT_PLAYERS === 1`

Reply:
It will be; could add a check for `players === 1` in the `do`.

Reply:
What is benefit off adding in the `do` vs in `completed()`?

Reply:
Currently in completed. Can change to do if you prefer

Reply:
This works. My initial though was the completed logic is elegant with only the one condition, but I'm cool with either :+1: